tools/mpremote/tests/README.md

```diff
@@ -3,12 +3,26 @@
 This directory contains a set of tests for `mpremote`.
 
 Requirements:
-- A device running MicroPython connected to a serial port on the host.
+- A device running MicroPython connected to a serial port on the host, with the filesystem mounted as `/`
```

Comment:
The tests currently work fine on ports such as stm32 where the filesystem is not mounted on `/` (they are mounted on `/flash`).  And they should continue to run OK on such ports/boards.

Reply:
verified tests pass across all ports for which I have hardware, including stm32